### ISSUE TITLE
Glitch in movie

### ISSUE THREAD

**Issue by doubleteapot** [KyleKun/one_second_diary#132]
I tried to string together my 231 days of videos and it freezes at 11 seconds, on a certain clip. If I skip ahead it has sound but is still frozen on that image. And on the time bar it says the movie is over one hour long when it should only be a few minutes.
Thanks in advance

**Comment by TerenceAbigail** [comment]
@KyleKun I think this might already be fixed with the changes in https://github.com/KyleKun/one_second_diary/pull/147. I am suspecting this is due to the audio track issue that was fixed in that PR

**Comment by doubleteapot** [comment]
Thanks for the update! I just tried it again, and it didn't freeze anymore, but the sound stopped working partway through, and the video still says it is way longer than it should be. I've just given up now and just linked the one second clips together on my computer instead so I can watch my 2023 video at last haha, luckily that is an option.

**Comment by TerenceAbigail** [comment]
@doubleteapot to be clear this isn't fixed in the currently released version of the app. The fix will be in a hopefully to be released version in the near future.

I was just flagging this to the project maintainer that we might have this issue fixed already in some other PR

**Comment by doubleteapot** [comment]
Ah I see, thanks for explaining!

**Comment by KyleKun** [comment]
> @KyleKun I think this might already be fixed with the changes in https://github.com/KyleKun/one_second_diary/pull/147. I am suspecting this is due to the audio track issue that was fixed in that PR

Yeah, tested with a couple different videos and didn't get the issue either, will do some more testing before releasing but we can close this now
